docs/src/main/asciidoc/spring-cloud-config.adoc

```diff
@@ -546,12 +546,17 @@ The following table describes configurable Vault properties:
 |profileSeparator
 |,
 
+|versioningEnabled
```

Comment:
I think I'd name it along the Vault K/V backend version. Right now, we assume the only change is versioning but we should stay open for the future and not introduce a possible ambiguity. Therefore, let's switch to `kvVersion` (`kvVersion=1`, `kvVersion=2`).